src/components/reports/formSteps/Report.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from "react";
+
+import FieldSet from "../FieldSet";
+import FormStepBtn from "../../UI/FormStepBtn";
+
+interface Props {
+  nextStep: () => void;
+  prevStep: () => void;
+}
```

Comment:
would it be worth having this as a shared interface that each step can use, extending it as necessary?